dev/bazel/config/cpudetect.cpp

```diff
@@ -14,7 +15,11 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include "cpp/daal/include/services/daal_defines.h"
```

Comment:
This line still is causing some trouble in our CI - lets try redefining the imported variables instead (add [this logic](https://github.com/oneapi-src/oneDAL/pull/2614/files#diff-830416203aa5bbb105889fc222326f2b0057677eb29abca3727a56e83c1d0b4bR32) instead of including daal_defines)

Reply:
okay, made those changes now